lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -147,3 +147,14 @@ bool CPUBackend::canDoIndexTypeDemotion(
       Kinded::Kind::SparseToDenseMaskNodeKind);
   return fromTy == ElemKind::Int64ITy && toTy == ElemKind::Int32ITy;
 }
+
+/// \returns an array of raw objects which are statically allocated and
```

Comment:
This is an override and probably just copies the comment from a parent method? If so, let's not repeat it here.

Reply:
Removed the comment. Done.